source/fuzz/transformation_add_dead_break.cpp

```diff
@@ -112,7 +112,7 @@ bool TransformationAddDeadBreak::IsApplicable(
     const TransformationContext& transformation_context) const {
   // First, we check that a constant with the same value as
   // |message_.break_condition_value| is present.
-  if (!fuzzerutil::MaybeGetBoolConstant(ir_context,
+  if (!fuzzerutil::MaybeGetBoolConstant(ir_context, transformation_context,
```

Comment:
It's sort of weird here having to pass in transformation_context but not the boolean to say that the id is relevant.

Reply:
We need a transformation context regardless of whether the id is relevant or not.